### ISSUE TITLE
reaching inside other run directories

### ISSUE THREAD

**Issue by robnagler** [radiasoft/sirepo#1950]
There needs to be a clearer way to get access to data from other simulations and run directories. Right now, it's informal, and makes assumptions that the directories are all shared on all nodes. This isn't going to be the case with NERSC or clusters.  For example, jspec.get_elegant_sim_list looks inside elegant's directories to find those simulations which have twiss parameters. There's been some clean up by explicitly coupling the names, but more needs to be done.

**Comment by robnagler** [comment]
See #2028

**Comment by bruhwiler** [comment]
@robnagler -- can this issue be closed?

**Comment by robnagler** [comment]
No, it's still a problem. We need to build an API that supports asking the db for the appropriate information.

**Comment by robnagler** [comment]
Break out into separate issues for each code.
